### DISCUSSION TITLE
Vercel deployment looks like it's in progress in Bitbucket but it's ignored with "Ignored Build Step" setting.

### DISCUSSION THREAD

**Discussion by mare95** [vercel/community#2917] [Help] [answered]
### Summary

Few days ago everything seemed fine but now when i create a PR in bitbucket, it looks like Vercel deployment is in progress. This is incorrect because this deployment is canceled by  "Ignored Build Step" setting in Vercel dashboard.

Everything works regarding ignoring the build except the fact that the job status icon being incorrect in bitbucket PR. It is very annoying to see "in progress" icon near Vercel job in bitbucket PR because it's automatically canceled and it should be a success icon (green check mark). So every time i see that icon i need to go to the Vercel to make sure everything is ok (build is actually canceled and there are no issues)
See attached images.

Do you know why this started happening?
<img width="881" alt="Screenshot 2023-06-19 at 12 12 15" src="https://github.com/vercel/community/assets/17355348/62f25a05-4b08-4ce5-b433-d1f2405d61e6">
<img width="950" alt="Screenshot 2023-06-19 at 12 17 37" src="https://github.com/vercel/community/assets/17355348/3e7e00b8-de63-4af1-8eb7-57b57e3f6cac">
<img width="103" alt="Screenshot 2023-06-19 at 12 09 57" src="https://github.com/vercel/community/assets/17355348/a8f2723b-d090-4c3d-acca-09e9f8871424">

Thanks a lot :) 

### Steps to Reproduce

- Connect your bitbucket repo with Vercel
- In ignored build settings section add `exit 0;`  which ignores the deployment.
- Create a PR and Vercel job icon will always be "blue clock" which means that the job is in progress but it's actually canceled immediately so it should return success status code.

**Comment by amyegan** [comment, marked as answer]
Hi @mare95! Thanks for sharing this feedback about the cancelled build status. 

If you want to completely disable builds so they don't even start, then you should can [`git.deploymentEnabled`](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#git.deploymentenabled). I think that option will work better for you.

**Reply by mare95** [reply]
Niceee! I couldn't find this config so i just decided to ignore every build that was triggered when committing to the branch.

Thanks @amyegan :)
